docs/pages/versions/unversioned/sdk/filesystem.mdx

```diff
@@ -128,13 +128,12 @@ You can upload files as blobs directly with `fetch` built into the Expo package:
 import { fetch } from 'expo/fetch';
 import { File } from 'expo-file-system';
 
-const src = new File(testDirectory, 'file.txt');
+const file = new File(testDirectory, 'file.txt');
```

Comment:
also, should `testDirectory` be `Paths.cache` here? Asking because there's a recent community PR: https://github.com/expo/expo/pull/39340/files do update this example.